ilc/server/tailor/fragment-hooks.js

```diff
@@ -18,11 +18,15 @@ function insertStart(stream, attributes, headers, index) {
             stream.write(
                 isAsync
                     ? `<!-- Async fragments are not fully implemented yet: ${ref.uri} -->`
-                    : `<link rel="stylesheet" href="${ref.uri}"${
-                        id !== undefined
-                            ? ` data-fragment-id="${id}"`
-                            : ''
-                        }>`
+                    : id
+                        ?
+                            '<script>(function(url, id){' + 
+                                `const link = document.head.querySelector('link[data-fragment-id="' + id + '"]');` +
+                                'if (link) {' +
+                                    `link.href = url;` +
+                                '}' +
+                            `})("${ref.uri}", "${id}");document.currentScript.remove();</script>`
```

Comment:
`document.currentScript` && `ChildNode.remove()` doesn't work in IE :(

Reply:
I think we can keep this script w/o removal if you won't be able to find a good solution quickly.

Reply:
I forgot we load polyfills near the end of the body
I decided to remove it now